vendor/config.py

```diff
@@ -1,7 +1,88 @@
 # App Settings
 from django.conf import settings
+from django import forms
+from django.db.models import TextChoices
+from django.contrib.sites.models import Site
+from django.core.exceptions import ObjectDoesNotExist
 from django.utils.translation import ugettext_lazy as _
 
+from siteconfigs.config import SiteConfigBaseClass
+from siteconfigs.models import SiteConfigModel
+
+class SupportedPaymentProcessor(TextChoices):
+    PROMO_CODE_BASE = ("base.PaymentProcessorBase", _("Default Processor"))
+    AUTHORIZE_NET = ("authorizenet.AuthorizeNetProcessor", _("Authorize.Net"))
+
+
+class PaymentProcessorForm(forms.Form):
+    processor = forms.CharField(label=_("Payment Processor"), widget=forms.Select(choices=SupportedPaymentProcessor.choices))
+
+    def __init__(self, *args, **kwargs):
```

Comment:
this init seems redundant, since you already set the label and widget in the field declaration

Reply:
yes it is :)

Reply:
but you already have set them, when you declared the field

Reply:
you are right